Storage/tests/Unit/StorageClientTest.php

```diff
@@ -352,6 +356,306 @@ public function requiresProjectIdMethods()
             ['createHmacKey', ['foo']]
         ];
     }
+
+    public static function getSuccessfulObjectsResponse()
```

Comment:
This should be private as well
```suggestion
    private static function getSuccessfulObjectsResponse()
```